sdk/bpf/scripts/install.sh

```diff
@@ -11,75 +11,78 @@ fi
 download() {
   set -e
   declare url="$1/$2/$3"
-  declare version=$2
   declare filename=$3
-  declare dirname=$4
-  declare progress=$5
-  declare cache_directory=~/.cache/"$version"
-  declare cache_dirname=$cache_directory/${dirname//:\//_}
-
-  link() {
-    set -e
-    ln -sf "$cache_dirname" "$dirname"
-    ln -sf "$cache_directory/$dirname-$version.md" "$dirname-$version.md"
-  }
-
-  if [[ -r $cache_dirname ]]; then
-    link
-    return 0
-  fi
-
-  declare args=(
-    "$url" -O "$filename"
-    "--progress=dot:$progress"
+  declare wget_args=(
+    "$url -O $filename"
+    "--progress=dot:giga"
     "--retry-connrefused"
     "--read-timeout=30"
   )
+  declare curl_args=(
+    "-L $url -o $filename"
+  )
+  wget_or_curl=$(
+    if hash wget 2>/dev/null; then
+      echo "wget ${wget_args[*]}"
+    elif hash curl 2>/dev/null; then
+      echo "curl ${curl_args[*]}"
+    fi
+  )
+  if [ -z "$wget_or_curl" ]; then
+    echo "Error: Neither curl nor wget were found" >&2
+    return 1
+  fi
   set -x
-  mkdir -p "$cache_dirname"
-  pushd "$cache_dirname"
-  if wget "${args[@]}"; then
+  if $wget_or_curl; then
     tar --strip-components 1 -jxf "$filename"
```

Comment:
Errors get eaten without an explicit return when in a function, regardless of global `set -e`:
```suggestion
    tar --strip-components 1 -jxf "$filename" || return 1
```